participants/IPFS-Found.md

```diff
@@ -0,0 +1,66 @@
+# <Project Name>
+
+Before submitting a PR containing your team's project information:
+
+- Make sure you rename this file to contain your project's name
+- Respond to add your response in each section below
+
+## (0) Project Name and Team Slack Handles
+
+- Project Name : IPFS Found
+- Slack Handles : IPFSUnion.cn
+
+## (1) Brief Project Description
+
+Based on Filecoin network, we develop an application platform for storage and retrieval to help users store various types of files, including text, picture, audio, video, etc. IPFS-Found supports storage of multiple data sizes. We aim to build more decentralized, stable and robust data storage platform.
+
+## (2) Link to Project UI
+
+Link : https://filecoin.in
```

Comment:
Is this a usable website?

Reply:
We improve some tips and it can be used now.